sycl/include/CL/sycl/detail/helpers.hpp

```diff
@@ -33,6 +33,12 @@ template <int Dims> class h_item;
 enum class memory_order;
 
 namespace detail {
+inline void memcpy(char *Dst, const char *Src, size_t Size) {
```

Comment:
Could you make `detail::memcpy` API similar to `std::memcpy` to avoid additional casts?

Reply:
Done